rids.service

```diff
@@ -4,7 +4,7 @@ Description=Remote Intrusion Detection System daemon
 [Service]
 User=root
 WorkingDirectory=/usr/local/sbin
-ExecStart=/usr/local/sbin/rids.py
+ExecStart="/usr/local/sbin/rids.py --eventlog_path=/var/rids/events.log"
```

Comment:
Nice to have: you might want to specify the location of the config here too, as documentation, so the host owner know where everything is by reading the service file.

Reply:
done